hpcgap/lib/vecmat.gi

```diff
@@ -610,13 +605,6 @@ InstallOtherMethod( \[\]\:\=,
       IsObject ],
     ASS_GF2MAT );
 
-InstallMethod( \[\]\:\=,
-    "for GF2 matrix",
-    [ IsGF2MatrixRep,
-      IsPosInt, IsPosInt,
-      IsObject ],
-    SET_MAT_ELM_GF2MAT );
```

Comment:
This is now redundant as we already do `InstallMethod(SetMatElm, ..., SET_MAT_ELM_GF2MAT);` later in the file.